{{cookiecutter.repo_name}}/manage.py

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python
 import os
+import pwd
 import sys
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{cookiecutter.package_name}}.settings.local")
+    os.environ.setdefault("DB_USER", pwd.getpwuid(os.getuid()).pw_name)
```

Comment:
What does this do?

Reply:
This is copied from what was previously in `local.py`. It will get the os user.

Reply:
Oh, fair enough.